shardingsphere-governance/shardingsphere-governance-context/src/main/java/org/apache/shardingsphere/governance/context/metadata/GovernanceMetaDataContexts.java

```diff
@@ -320,6 +322,22 @@ public synchronized void renew(final PrimaryStateChangedEvent event) {
         }
     }
     
+    /**
+     * Renew global rule configurations.
+     *
+     * @param event global rule configurations changed event
+     */
+    @Subscribe
+    public synchronized void renew(final GlobalRuleConfigurationsChangedEvent event) {
+        Collection<RuleConfiguration> newGlobalConfigs = event.getRuleConfigurations();
+        if (!newGlobalConfigs.isEmpty()) {
+            ShardingSphereRuleMetaData newGlobalRuleMetaData = new ShardingSphereRuleMetaData(newGlobalConfigs,
+                    ShardingSphereRulesBuilder.buildGlobalRules(newGlobalConfigs, metaDataContexts.getMetaDataMap()));
+            metaDataContexts = new StandardMetaDataContexts(
+                    metaDataContexts.getMetaDataMap(), newGlobalRuleMetaData, metaDataContexts.getExecutorEngine(), metaDataContexts.getUsers(), metaDataContexts.getProps());
```

Comment:
Should `metaDataContexts.getUsers()` be retrieved from newGlobalConfigs?

Reply:
because `user `  had save into global authority rule ,so field `ShardingSphereUsers ` in  `StandardMetaDataContexts` will be remove at the next step

Reply:
> Should `metaDataContexts.getUsers()` be retrieved from newGlobalConfigs?

because `user` had save into global authority rule ,so field `ShardingSphereUsers` in `StandardMetaDataContexts ` will be remove at the next step

Reply:
Hi @huanghao495430759 ,
Please look at #10310, @zhujunxxxxx is on the way.

Reply:
OK, I see.